perses/app/relative_point_mutation_setup.py

```diff
@@ -103,7 +111,8 @@ def __init__(self,
                  apo_box_dimensions=None,
                  flatten_torsions=False,
                  flatten_exceptions=False,
-                 repartitioned_endstate=None,
+                 vanilla=True,
```

Comment:
Can we use a more informative name than `vanilla`? And instead of selecting "generate the unmodified form" if `True`, perhaps we could instead use a flag for generating the modified form, giving it a more informative name?

Reply:
The flag below this line is `repartitioned=True`, which allows generating the repartitioned htf. I want to allow the user to only generate the repartitioned htf or the vanilla htf or both, but I could change the flags to be `generate_vanilla_htf` and `generate_repartitioned_htf` if you prefer.

Reply:
Ah, I see! Yes, that would be much clearer, though perhaps we could use clearer names?

Perhaps `generate_unmodified_hybrid_topology_factory` and `generate_rest_capable_hybrid_topology_factory`.
They're long, but they're unambiguous.

Reply:
Fixed